### ISSUE TITLE
Prompt to restore old password if Steam responds with LOGIN_FAILED 

### ISSUE THREAD

**Issue by reblws** [steevp/UpdogFarmer#14]
Right now it forgets and deletes my password if Steam responds with LOGIN_failed. Is it possible to add a prompt that lets me restore my old password instead of deleting it straight away?

Thank for great app.

**Comment by NewOTAKUz** [comment]
Gg

**Comment by SpiceWareX** [comment]
Yes, this is possible through shared_secret which is a code you can find through https://github.com/SteamTimeIdler/stidler/wiki/Getting-your-%27shared_secret%27-code-for-use-with-Auto-Restarter-on-Mobile-Authentication

If there's a new feature to auto restart the App, this code will be needed. Don't use this code on untrusted websites because this code gives access to Steam Guard of your account indirectly.
